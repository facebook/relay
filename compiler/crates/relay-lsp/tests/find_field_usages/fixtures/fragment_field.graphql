fragment TestLinkedFieldFragment on User {
  name
}

%arguments%
User name
