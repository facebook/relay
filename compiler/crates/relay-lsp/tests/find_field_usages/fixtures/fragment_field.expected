==================================== INPUT ====================================
fragment TestLinkedFieldFragment on User {
  name
}

%arguments%
User name
==================================== OUTPUT ===================================
("TestLinkedFieldFragment", fragment_field.graphql:45:49)
